src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -122,7 +122,7 @@ def tab(id, text, opts)
     opts[:package] = @package.to_s if @package
     opts[:project] = @project.to_s if @project
     link_opts = { id: "tab-#{id}" }
-    if (@current_action.to_s == opts[:action].to_s && @current_controller.to_s == opts[:controller].to_s) || opts[:selected]
+    if (action_name == opts[:action].to_s && controller_name == opts[:controller].to_s) || opts[:selected]
```

Comment:
the `@current_action` variable is used in the tests: https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/helpers/webui/webui_helper_spec.rb#L222

Test need to be fixed :wink:

Reply:
@Ana06 Done :wink: